dbt/include/snowflake/profile_template.yml

```diff
@@ -2,33 +2,33 @@ fixed:
   type: snowflake
 prompts:
   account:
-    hint: 'https://<this_value>.snowflakecomputing.com'
+    hint: 'What is your account id for Snowflake? [https://<this_value>.snowflakecomputing.com]'
   user:
-    hint: 'dev username'
+    hint: 'What is your personal user login for Snowflake?'
   _choose_authentication_type:
     password:
       password:
-        hint: 'dev password'
+        hint: 'What is your password for Snowflake?'
         hide_input: true
     keypair:
       private_key_path:
-        hint: 'path/to/private.key'
+        hint: 'What is the path/to/private.key for Snowflake authentication?'
```

Comment:
I like that we're making these more "human" hints, but then the path example should probably be updated to reflect that grammatically; e.g. `path/to/the/private.key`.

Reply:
Resolved via https://github.com/dbt-labs/dbt-snowflake/pull/729/commits/58ce8f2838a9f9b91c11c2cb3d92aa568c09d25a